packages/core/package.json

```diff
@@ -40,8 +40,8 @@
     "heap-js": "2.1.6",
     "leveldown": "6.1.0",
     "levelup": "5.1.1",
-    "libp2p": "0.35.8",
-    "libp2p-kad-dht": "0.27.6",
+    "libp2p": "0.36.2",
+    "libp2p-kad-dht": "0.28.6",
     "libp2p-mplex": "0.10.7",
```

Comment:
shouldn't you update mplex as well?

Reply:
As discussed, didn't update this because newer version have moved to use a different libp2p interface library.